tests/conftest.py

```diff
@@ -13,8 +13,13 @@
 
 from zennit.attribution import identity
 from zennit.core import Composite, Hook
-from zennit.composites import COMPOSITES, NameMapComposite, LayerMapComposite, SpecialFirstLayerMapComposite
+from zennit.composites import COMPOSITES
 from zennit.composites import EpsilonGammaBox
+from zennit.composites import LayerMapComposite
+from zennit.composites import MixedComposite
+from zennit.composites import NameLayerMapComposite
+from zennit.composites import NameMapComposite
+from zennit.composites import SpecialFirstLayerMapComposite
```

Comment:
A little bit of a style-change in the imports, but I'll allow it.

Reply:
Yes, PEP8 doesn't really specify the definitive way how to write multiline imports, so I followed this one: https://github.com/asottile/reorder_python_imports